master/static/migrations/20230612112824_add-rp-workspace-bindings.tx.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE rp_workspace_bindings ( 
+    workspace_id INT NOT NULL, 
+    pool_name TEXT NOT NULL, 
+    valid BOOLEAN NOT NULL,
```

Comment:
nit: the ERD says:

> If a resource pool no longer exists in the cluster configuration, the workspace <> resource pool 
> bindings are disabled, but remain in the table. Warnings should be returned to the cluster admin/logs. 
> - Desired UI --> grayed out bindings 

I'm assuming that's what this `valid` bool represents. If that's the case, it needs to be passed to the web in some way in order to achieve the desired UI

Reply:
Good point, looking back at it though I don't think it makes sense because it specifically calls out the case where a resourcepool is renamed, in which case it won't show up on the cluster management page. We'll log it when it happens in the master logs